Copy to SD Card root directory to update/Language Packs/language_uk.ini

```diff
@@ -202,8 +202,8 @@ label_move_speed:Швидкість (X Y Z)
 label_knob_led:Підсвічування енкодера
 label_knob_led_idle:Простій підсвічування енкодера
 label_m0_pause:Пауза по команді M0
-label_send_start_gcode:G-код перед друком
-label_send_end_gcode:G-код після друку
+label_send_start_gcode:Розпочати G-код перед друком
```

Comment:
I don't think these are correct. You don't "begin" or "end" a G-code.
It toggles sending start/end g-code On or Off.

Reply:
English version:
`label_send_start_gcode:Start Gcode before print
label_send_end_gcode:End Gcode after print`

Reply:
I think "Start Gcode" means the gcode, which is exected on the start.
Same with the "End Gcode" - gcode, which is executed on the end.